openproblems/tasks/chromatin_potential/methods/beta.py

```diff
@@ -69,33 +87,38 @@ def _get_annotation(adata, retries=3):
     )
 
 
-def _atac_genes_score(adata, top_genes=500, threshold=1):
+def _atac_genes_score(adata, top_genes=2000, threshold=1, method="beta"):
     """Calculate gene scores and insert into .obsm."""
     import pybedtools
 
     # get annotation for TSS
     _get_annotation(adata)
     # basic quality control
-    if adata.shape[0] >= 600:
+    if adata.shape[0] >= 10000:
```

Comment:
why 10000? can we go for a more elegant solution? (like, if after filtering there are less than x cells, say 100 cells, then don't filter)

Reply:
Yes, I add a more elegant solution with a minimum cell number of 100.